.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Run Unit Tests
+
+on: [push]
```

Comment:
Should this run on pull requests as well?
https://github.com/DataBiosphere/terra-cloud-resource-lib/blob/7abed537fa4a6d02189dde8b78d62aff6ebd100d/.github/workflows/build-and-test.yaml#L6-L10